### ISSUE TITLE
Migration generation doesn't detect drop tables only changes

### ISSUE THREAD

**Issue by adayth** [CakeDC/migrations#210]
I tried everything to generate a migration after dropping two tables of my database without success. 
I always got "No database changes detected." message

After adding a dummy field to another table, I was be able to generate the migration.

I was using github 2.3.6 tagged version.

**Comment by steinkel** [comment]
Aday,
The tables dropped were related to a Model present in your application?
Tables were present in the schema file before dropping?

**Comment by adayth** [comment]
The tables where HABM *companies_users* and a dummy table called *debugs* that I forget to delete. *CompaniesUser* and *Debug* models existed in the app. I reviewed the commit and I don't know if I deleted these models classes before or after trying to create the migration.

To me the most important is that after adding a dummy field to another table, the migration was generated perfectly and I just needed to remove lines related to that dummy field.
